src/elf/reloc.rs

```diff
@@ -183,12 +189,9 @@ macro_rules! elf_rela_std_impl { ($size:ident, $isize:ty) => {
 
             pub fn from_fd(fd: &mut File, offset: usize, size: usize) -> Result<Vec<Rela>> {
                 let count = size / SIZEOF_RELA;
-                let mut bytes = vec![0u8; size];
+                let mut relocs = vec![Rela::default(); count];
```

Comment:
This is also another dubiously useful function, a holdover before I started memmaping in dryad, iirc